src/drivers/fluid_wasapi.c

```diff
@@ -672,7 +680,15 @@ static void fluid_wasapi_foreach_device(fluid_wasapi_devenum_callback_t callback
         IMMDevice_Release(dev);
     }
 
-cleanup:
+
+cleanup1:
+
+    if (!com_was_initialized)
+    {
+        CoUninitialize();
```

Comment:
`CoUninitialize` must be called at the very end. When you call it here, other COM objects are still alive, which causes a segfault.